app/components/audit_log/row.js

```diff
@@ -204,6 +246,17 @@ export default Relay.createContainer(AuditLogRow, {
             ...on Pipeline {
               name
             }
+            ...on Team {
+              name
+            }
+            ...on TeamMember {
+              team { name }
+              user { name }
+            }
+            ...on TeamPipeline {
+              team { name }
+              pipeline { name }
```

Comment:
I know it doesn’t affect much but for the sake of future diffs can we keep each query property on a separate line? 😄